File: NetFramework/TouchToolkit/VB/TouchToolkitDemo/MultiScaleImagePages/MultiScaleImageDemo.vb
Imports System.Collections.Generic
Imports System.ComponentModel
Imports System.Drawing
Imports System.Data
Imports System.Linq
Imports System.Text
Imports System.Windows.Forms

Namespace MultiScaleImagePages
    Partial Public Class MultiScaleImageDemo
        Inherits DemoBase
        Public Sub New()
            InitializeComponent()

            Me.Title = "C1MultiScaleImage"
            Me.Description = "C1MultiScaleImage can show image with better quality in zoom. This page shows difference between PictureBox and C1MultiScaleImage. " & vbLf & " -Please try zoom in the following zoompanel by touch."

            zoomFactorLabel.Text = "ZoomFactor:" & (gcZoomPanel1.ZoomFactor * 100).ToString() & "%"
            AddHandler gcZoomPanel1.ZoomFactorChanged, AddressOf gcZoomPanel1_ZoomFactorChanged
        End Sub

        Private Sub gcZoomPanel1_ZoomFactorChanged(sender As Object, e As EventArgs)
            zoomFactorLabel.Text = "ZoomFactor:" & (gcZoomPanel1.ZoomFactor * 100).ToString() & "%"
        End Sub
    End Class
End Namespace

File: NetFramework/TouchToolkit/CS/TouchToolkitDemo/MultiScaleImagePages/MultiScaleImageDemo.cs
using System;
using System.Collections.Generic;
using System.ComponentModel;
using System.Drawing;
using System.Data;
using System.Linq;
using System.Text;
using System.Windows.Forms;

namespace TouchToolkitDemo.MultiScaleImagePages
{
    public partial class MultiScaleImageDemo : DemoBase
    {  
        public MultiScaleImageDemo()
        {
            InitializeComponent();

            this.Title = "C1MultiScaleImage";
            this.Description = "C1MultiScaleImage can show image with better quality in zoom. This page shows difference between PictureBox and C1MultiScaleImage. \n -Please try zoom in the following zoompanel by touch.";
            
            zoomFactorLabel.Text = "ZoomFactor:" + (C1ZoomPanel1.ZoomFactor * 100).ToString() + "%";
            C1ZoomPanel1.ZoomFactorChanged += C1ZoomPanel1_ZoomFactorChanged;
        }

        void C1ZoomPanel1_ZoomFactorChanged(object sender, EventArgs e)
        {
            zoomFactorLabel.Text = "ZoomFactor:" + (C1ZoomPanel1.ZoomFactor*100).ToString() + "%";
        }
    }
}
